pydecred/rpc.py

```diff
@@ -60,12 +60,183 @@ def getBlockchainInfo(self):
+        height, all requests will return an error response in order to avoid
+        serving stale data.
+
+        Args:
+            address (str): The Decred address to search for
+            verbose (int): Optional. default=1. Specifies the transaction is returned as a JSON
+                object instead of hex-encoded string.
+            skip (int): Optional. Default=0. The number of leading transactions to leave out of the
+                final response.
+            count (int): Optional. Default=100. The maximum number of transactions to return.
+            vinextra (int): Optional. Default=0. Specify that extra data from previous output will be
+                returned in vin.
+            reverse (bool): Optional. Default=False. Specifies that the transactions should be returned
+                in reverse chronological order.
+            filteraddrs (list(str)): Optional. Default=[]. Only inputs or outputs with matching
+                address will be returned.
+
+        Returns:
+            list(SearchRawTransactionResult): The SearchRawTransactionResults.
+        """
+        return [
+            SearchRawTransactionsResult.parse(rawTx)
+            for rawTx in self.call(
+                "searchrawtransactions",
+                address,
+                verbose,
+                skip,
+                count,
+                vinextra,
+                reverse,
+                filteraddrs,
+            )
+        ]
+
+    def sendRawTransaction(self, msgTx, allowhighfees=False):
+        """
+        Submits the serialized, hex-encoded transaction to the local peer and
+        relays it to the network.
+
+        Args:
+            msgTx (object): msgtx.MsgTx signed transaction.
+            allowhighfees (bool): Optional. Default=False. Whether or not to allow insanely high fees
+                (dcrd does not yet implement this parameter, so it has no effect).
+
+        Returns:
+            bytes-like: The hash of the transaction.
+        """
+        txid = self.call("sendrawtransaction", msgTx.txHex(), allowhighfees)
+        return reversed(ByteArray(txid))
+
+    def setGenerate(self, generate, genproclimit=-1):
+        """
+        Set the server to generate coins (mine) or not.
+
+        Args:
+            generate (bool): Use True to enable generation, False to disable it.
+            genproclimit (int): Optional. Default=-1. The number of processors (cores) to limit
+                generation to or -1 for default.
+        """
+        self.call("setgenerate", generate, genproclimit)
+
+    def stop(self):
+        """
+        Shutdown dcrd.
+
+        Returns:
+            str: 'dcrd stopping.'
+        """
+        return self.call("stop")
+
+    def submitBlock(self, hexblock, options={}):
+        """
+        Attempts to submit a new serialized, hex-encoded block to the network.
+
+        Args:
+            hexblock (str): Serialized, hex-encoded block.
```

Comment:
~~hexblock~~ -> hexBlock

If we ever implement `MsgBlock`, we will switch to that. `rpcclient.Client` uses a `dcrutil.Block`, but really only uses the underlying `MsgBlock` anyway.